src/compiler/orcv2.jl

```diff
@@ -258,6 +286,13 @@ function add!(mod)
         replace_uses!(f, ptr)
         Compiler.eraseInst(mod, f)
     end
+
+    # rename non-special global constants that
+    # have external linkage (modified by the ccall
```

Comment:
alternate solution, can we just mark all defined symbols during the lto import phase as internal?

we should already do so for functions

Reply:
x/ref https://github.com/EnzymeAD/Enzyme.jl/blob/c009fa593821cbd9017450a6721259f7dbc6ebf1/src/compiler/validation.jl#L596